ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst

```diff
@@ -6,22 +6,28 @@
 Scatter Plots
 =============
 
-Atlanta, Georgia, which has a latitude of 33 degrees, is not included in
-this weather dataset. If you were asked to predict average January
+Charleston, South Carolina, which has a latitude of 33 degrees, is not included 
+in this weather dataset. If you were asked to predict average January
 temperatures there, you might make a good guess and say 52 degrees F,
 because that is the mean temperature for southern cities with a latitude
-below 35. The problem with this prediction is that it is the same for
+below 35. This was found using the bar chart that you saw in the introduction:
+
+.. image:: figures/scatter3.png
```

Comment:
This image isn't added in this PR, where is it?

Reply:
scatter3.png already existed. It was used in a different section, and I didn't take it out so it should still be in figures.

Reply:
Ok LGTM. Once pavement.py is removed from the PR, we can merge.

Reply:
@kaitlyn-zeichick

Reply:
Reminder to get this fixed @kaitlyn-zeichick @lindseyc

Reply:
Pavement.py was removed. Did I do it incorrectly?

Reply:
Just added it back!